service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -167,8 +167,8 @@ public String getMostRecentUploadedOrderDocumentUrl(final List<Element<Generated
     }
 
     public void removeOrderProperties(Map<String, Object> caseData) {
-        Arrays.asList(GeneratedOrderKey.values(), GeneratedEPOKey.values())
-            .forEach(caseData::remove);
+        Arrays.stream(GeneratedEPOKey.values()).forEach(e -> caseData.remove(e.getKey()));
```

Comment:
Would probably change the name from `e` to something more useful

Reply:
Yeah bad name. Will change